package.json

```diff
@@ -120,7 +120,7 @@
     "jest": "28.1.1",
     "jest-environment-jsdom": "28.1.1",
     "jest-fetch-mock": "^3.0.3",
-    "lerna": "^3.22.1",
+    "lerna": "^6.4.0",
```

Comment:
@itrulia is there reason to keep 3.22 still with nx?

Reply:
Kept it at 3.22 because I dont know if any breaking changes affected us. Else we can use both.
The integration between lerna and nx doesn't really affect us since we don't use the lerna build/test commands.

Reply:
So let's remove it if nx does its own dep.